.github/workflows/cla-assistant.yaml

```diff
@@ -1,11 +1,11 @@
+name: "CLA bot"
+
 on:
   issue_comment:
     types: [created]
   pull_request_target:
     types: [opened,closed,synchronize]
 
-name: "CLA bot"
```

Comment:
why remove this name?

Reply:
it's only moved to the top of file ^^ pedantic me on a Friday :)

Reply:
somehow missed that in the diff... makes sense!